### PR TITLE
AR-771 Open clips section crash after ui language change

### PR DESCRIPTION
**Pull request by guya**

### CODE REVIEW

File: src/components/Sections/Programs/tabs/Clips/List.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const renderUnit = (unit, t) => {
   const { clips, relatedItems }
           = renderUnitHelper.commonRenderUnitForClips(unit, t);
 
-  return (
+  return unit ? (
```
Comment (gshilin) [inline src/components/Sections/Programs/tabs/Clips/List.js:12]: 1. Try to return null instead if empty string. Will it work?
2. Change ?: to 
   if (!unit) {
      return null;
   }

   <jsx>...

File: src/helpers/renderUnitHelper.js [hunk 2]
```diff
@@ -39,33 +39,33 @@ export const renderUnitFilmDate = (unit, t) => {
   return filmDate ? renderFilmDate(filmDate) : null;
 };
 
-export const renderUnitNameAsListItem = unit => 
-  <List.Item key={unit.id} as={Link} to={canonicalLink(unit)}>
-    {unit.name || NO_NAME}
+export const renderUnitNameAsListItem = unit =>
+  <List.Item key={unit?.id} as={Link} to={canonicalLink(unit)}>
```
Comment (gshilin) [inline src/helpers/renderUnitHelper.js:43]: That's not correct to have key with value `undefined`. Better return `null` if unit is undefined

File: src/components/Sections/Programs/tabs/Clips/List.js (hunk 1 above)
Comment (guya) [reply src/components/Sections/Programs/tabs/Clips/List.js:12]: ok. it works

File: src/helpers/renderUnitHelper.js (hunk 2 above)
Comment (guya) [reply src/helpers/renderUnitHelper.js:43]: ok

File: src/components/Sections/Programs/tabs/Clips/List.js (hunk 1 above)
Comment (gshilin) [reply src/components/Sections/Programs/tabs/Clips/List.js:12]: null means for React: "do not render it"

Comment (guya) [reply src/components/Sections/Programs/tabs/Clips/List.js]: ok. good

**Pull request merged**